src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${name}`}</h2>
+    {age && <p className="Person__age">{`I am ${age}`}</p>}
+    <p className="Person__partner">
+      { isMarried
+        ? `My ${sex === 'm' ? 'wife\'s' : 'husband\'s'} name is ${partnerName}`
+        : 'I am not married'
+      }
```

Comment:
```suggestion
      { 
        isMarried
          ? `My ${sex === 'm' ? 'wife\'s' : 'husband\'s'} name is ${partnerName}`
          : 'I am not married'
      }
```

Reply:
according to all other checklists, it's ok, it also looks easier for me